<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contact Book</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget_databeans">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>171</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_OperateContact">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>371</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Operate Contact</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_operateContact_Name">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Unit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_operateContact_Unit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Telephone</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_operateContact_Telephone">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Mobile Phone</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_operateContact_MobilePhone">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_AddContact">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ModifyContact">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Modify</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_DeleteContact">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ListContact">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>List</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SearchContact">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>170</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ImportContact">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>220</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Import</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Note</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_operateContact_Note">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_ContactBook">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Contact Book:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>371</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Contact Info</string>
    </property>
    <widget class="QLabel" name="label_contact_info">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>361</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>587</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
